api/src/main/java/com/velocitypowered/api/event/proxy/ProxyBoundEvent.java

```diff
@@ -0,0 +1,25 @@
+package com.velocitypowered.api.event.proxy;
+
+import com.google.common.base.Preconditions;
+import java.net.InetSocketAddress;
+
+/**
+ * This event is fired by the proxy after the proxy starts accepting connections.
```

Comment:
As presently used, this will in fact fire after _any_ bind (this includes the query protocol and proxy reloads). Please update the documentation - personally, `ListenerBoundEvent` or similar would be a better name.

Reply:
I believe it only fires when the Minecraft port is bind, not query. But I'll update the documentation and the name.

Reply:
Should I also rename ProxyCloseEvent to something like `ListenerCloseEvent`, which fires after each listener is closed?

Reply:
Yeah, my bad - it won't fire for query. However, it probably should fire for query too. I do think a `ListenerCloseEvent` is better than a `ProxyCloseEvent`, which means reloads could be accommodated (say for a hypothetical dynamic DNS plugin updating a port).

Reply:
Would separate events be better (e.g. `ListenerBoundEvent` and `QueryListenerBoundEvent`) or some sort of field indication whether the listener is for query?